src/isolate-duplicates/index.js

```diff
@@ -1,3 +1,38 @@
-function isolateDuplicates(text) {}
+function isolateDuplicates(input) {
+  if (input === undefined) {
+    throw '';
```

Comment:
Prefer to `throw Error(...)` rather than strings. Helps preserve the stack trace for debugging.